objects/scene-objects/list.md

```diff
@@ -98,22 +103,50 @@ body {
 ```
 {% endtab %}
 
-{% tab title="Result" %}
-![](../../.gitbook/assets/audiobook-list.gif)
-{% endtab %}
 {% endtabs %}
 
 ### Js File
-The `Js File` **attribute** is a *JavaScript programming language* file that allows us to create a dynamic and interactive experience for **List**. The `Js File` can be created by right-clicking in the **Asset Manager** and selecting `Create asset` &gt; `JavaScript File`. 
+The `Js File` **attribute** is a *JavaScript programming language* file that allows us to add complex features and also create dynamic and interactive experiences for **List** **Objects**. 
 
+The `Js File` can be created by right-clicking in the **Asset Manager** and selecting `Create asset` &gt; `JavaScript File`. 
 For more information about JavaScript. Please visit the following website: [JavaScript programming language](https://developer.mozilla.org/en-US/docs/Web/JavaScript).
 
-The scenario below describes how *Javascript* can be us
+In the illustration below, we add user interaction to our audio-book **List** example using the `Js File` **Attribute**.
+
+{% tabs %}
+
+{% tab title="Result" %}
+
+![](../../.gitbook/assets/objects/scene-objects/list/replaceAudioBook.gif)
+
+{% endtab %}
+
+{% tab title="JavaScript" %}
+```JS
+function replaceBook(){
+    let bookName = prompt('Enter book name');
+    let author = prompt('Enter author')
+    let index = prompt('Index of book you would like to replace')
+
+    const item = document.getElementById('list').children[index];
+    
+    const newItem = "<h2>" + bookName + " by " + author + "</h2>";
+
+    if (item) {
+      item.innerHTML = newItem;
+      item.style.color = "#ADD8E6";
+    }
+}
+
+replaceBook();
+```
+{% endtab %}
+{% endtabs %}
```

Comment:
I assume this all works? I can't check is all.

Reply:
it does